Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -122,6 +163,28 @@
         "ServerCallFailed": "HTTP call to Jenkins server failed.",
         "ServerCallErrorCode": "HTTP response code: %d",
         "ArtifactSuccessfullyDownloaded": "Successfully downloaded artifact to %s",
-        "ArtifactDownloadUrl": "Downloading artifacts from: %s"
+        "ArtifactDownloadUrl": "Downloading artifacts from: %s",
+        "DownloadingCommitsAndWorkItems": "Downloading commits and work items",
+        "JenkinsDownloadingChangeFromCurrentBuild": "Cannot find start build, will be fetching commits and workitems from build used to download artifacts only: %s",
+        "DownloadingJenkinsChangeBetween": "Downloading commits and workitems between build %s to %s",
+        "GettingCommitsFromSingleBuild": "Downloading commits associated with the build %s",
+        "DownloadingContentFromJenkinsServer": "Downloading content form Jenkins server:%s with strict SSL:%s",
+        "WritingCommitsTo": "Writing commits to file %s",
+        "UploadingAttachment": "uploading %s as attachment",
+        "SuccessfullyUploadedCommitsAttachment": "Successfully uploaded commits attachment",
+        "GetCommitMessages": "Getting commit messages from commits",
+        "GetCommitMessagesFailed": "Getting commits message failed with an error %s",
+        "CouldNotWriteToFile": "could not save content to file. Failed with an error %s",
+        "DownloadingWorkItemsFromSingleBuild": "Downloading workitems associated with the build %s",
+        "WritingWorkItemsTo": "Writing workitems to %s",
+        "SuccessfullyUploadedWorkItemsAttachment": "Successfully uploaded workitem attachment",
+        "FindBuildIndex": "Trying to find the build's index",
+        "FoundBuildIndex": "Found startIndex %s and endIndex %s",
+        "JenkinsArtifactDetailsParsingError": "Parsing ArtifactDetails failed with an error %s",
+        "SuccessfullyDownloadedCommitsAndWorkItems": "Commits and WorkItems Downloaded successfully",
+        "CommitsAndWorkItemsDownloadFailed": "Downloading Jenkins Commits and WorkItem failed with an error: %s",
+        "InvalidBuildIndex": "Invalid build index found",
```

Comment:
Seems like an unfriendly user message or is it only for debugging? For all the strings, please review if these are user facing messages or debug messages. Debug messages don't have to be localized. You can use tl.debug() for those.

Reply:
Changed to generic "Cannot find builds". There are some debug message above to indicate exact issue found.